### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by pierromond** [Universite-Gustave-Eiffel/NoiseModelling#311] [Ideas]
Just a free discussion about what you would like to see in the next release!

**Comment by arlind89** [comment]
It would be interesting to see NoiseModelling integrated with Rhino-Grasshopper, or Blender which is an open-source modelling software. Simulations until now are mostly used to evaluate and validate current state conditions, e,g, of a project or at city-level. 
When simulations are used as a design tool, they can inform/support the design process, not only just to report the performance or the urban planning zoning decisions. It would change a bit the role of how simulations are used. GIS-CAD is not yet well integrated, unfortunately, and it is not as simple from a designer's perspective the current workflows.
I hope I was a bit clear in my suggestion.

**Reply by ioannides-dB** [reply]
I share the same thoughts on this i.e. Currently a very powerful evaluation tool, but for implementing mitigation measures, as usually is in my case, both professionally and from an educational/research point of view, it would be very useful to have it integrated within a modeling environment.
